beacon-chain/sync/service.go

```diff
@@ -0,0 +1,134 @@
+package sync
+
+import (
+	"context"
+	"hash"
+
+	"github.com/prysmaticlabs/prysm/beacon-chain/types"
+	"github.com/sirupsen/logrus"
+)
+
+var log = logrus.WithField("prefix", "sync")
+
+// Service is the gateway and the bridge between the p2p network and the local beacon chain.
```

Comment:
s/Service/Sync service?

Reply:
I figure it's okay since an importer would prefre to see `sync.Service` rather than `sync.SyncService`

Reply:
You are right. please ignore my earlier comment 😅